### ISSUE TITLE
Still not compatible with laravel 9

### ISSUE THREAD

**Issue by amiralidevmoon** [beyondcode/laravel-websockets#956]
Hi whenever I'm trying to install the package it is throwing error in laravel 9

```
Root composer.json requires beyondcode/laravel-websockets ^1.12 -> satisfiable by beyondcode/laravel-websockets[1.12.0, 1.x-dev].

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires beyondcode/laravel-websockets ^1.13 -> satisfiable by beyondcode/laravel-websockets[1.13.0, 1.x-dev].
    - beyondcode/laravel-websockets[1.13.0, ..., 1.x-dev] require guzzlehttp/psr7 ^1.5 -> found guzzlehttp/psr7[1.5.0, ..., 1.x-dev] but the package is fixed to 2.1.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require beyondcode/laravel-websockets:*" to figure out if any version is installable, or "composer require beyondcode/laravel-websockets:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by erikn69** [comment]
`composer require "beyondcode/laravel-websockets:dev-master" -W`

**Comment by TimVanHerwijnen** [comment]
Running `composer require "beyondcode/laravel-websockets:dev-master"` without the `-W` worked for me.
After that I got error `Class "Doctrine\DBAL\Driver\AbstractMySQLDriver" not found` when migrating.
This was fixed by running `composer require doctrine/dbal`

**Comment by mpociot** [comment]
This is now fixed in 1.13.1 if you don't want to use the master branch
